.gitmodules

```
@@ -20,3 +20,9 @@
 [submodule "vitetris"]
 	path = vitetris
 	url = https://github.com/pulp-platform/vitetris.git
+[submodule "u-boot"]
+	path = u-boot
+	url = https://github.com/eyssartk/u-boot
```

Great, let's get this moved into OpenHW before merging.
```suggestion
	url = https://github.com/eyssartk/u-boot
```

---

Done !